src/coreclr/nativeaot/Runtime/unix/UnixContext.cpp

```diff
@@ -365,69 +365,9 @@ bool InitializeUnwindContextAndCursor(REGDISPLAY* regDisplay, unw_cursor_t* curs
     return true;
 }
 
-// Update context pointer for a register from the unw_cursor_t.
-static void GetContextPointer(unw_cursor_t *cursor, unw_context_t *unwContext, int reg, PTR_UIntNative *contextPointer)
-{
-    unw_save_loc_t saveLoc;
```

Comment:
I suspect of this is dummy move. I probably should use `unw_get_reg` instead of `unw_get_save_loc` to restore registry data.

Reply:
Now I see that WASM also do not have this function. Is this mean that it's gone for good?

Reply:
So appears that new version of libUnwind is also used for WASM, or `unw_get_save_loc` is undesired operation. I remove that defines for symmetry, and if this is desired function, I may have to make that function account for WASM too.